README.md

```diff
@@ -1,25 +1,82 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Budget app
 
-Things you may want to cover:
+> This app is all about budget and money. Users can create a catagory with the transactions the money spent on each transaction will be shown in the catagory.
 
-* Ruby version
+## Built With :hammer_and_wrench:
 
-* System dependencies
+- Major languages: Ruby
+- Frameworks: Ruby on Rails
+- Technologies used: Rubocop, gem, Postgresql
 
-* Configuration
+## Live Demo (if available)
 
-* Database creation
+[Live Demo Link](https://floating-shelf-06058.herokuapp.com/)
 
-* Database initialization
+## Setup
 
-* How to run the test suite
+To use this project:
 
-* Services (job queues, cache servers, search engines, etc.)
+- Open terminal
+- Clone it in your directory using
+  https://github.com/Alzubair98/budget_app.git
+- Then cd budget-app
 
-* Deployment instructions
+## Create, drop and, reset a database
 
-* ...
-# budget_app
+- rails db:create
+- rails db:drop
+- rails db:reset
+
+## Starting up the webserver
+
+- rails s # or
+- rails server
+
+## Generate schema
+
+- rails db:migrate
+
+## Generate model and controller specs
+
+-rails generate rspec:model ModelName
+-rails generate rspec:controller ControllerName
+
+## Run specs or tests
+
+- bin/rails spec or rspec
+
+## Authors :writing_hand: :busts_in_silhouette:
+
+:man_technologist: **Alzubair Alqaraghuli** :writing_hand:
+
+- GitHub: [@Alzubair98](https://github.com/Alzubair98)
+- Twitter: [@FitZubair](https://twitter.com/FitZubair)
+- LinkedIn: [@Alzubair Alqaraghuli](https://www.linkedin.com/in/alzubair-alqaraghuli-272918233/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a star ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration by (designer)[https://www.behance.net/gregoirevella]
+- etc
+
+## video
+
+- check this (link)[https://drive.google.com/drive/folders/1e5z4YFoYFhpK_oCVQ0g1wnITClikUGVA?usp=sharing]
```

Comment:
Good job giving access to your presentation video as mentioned by the previous reviewer. So, I noticed that it does match the required features as mentioned in the project's requirements and this should not be marked by the previous reviewer as you didn't give access. Make sure your video matches the below features:
  - The maximum length of the video is 5 minutes.
  - It should contain a highlight about a piece of the code you find interesting or you are proud of making.

Reply:
done